home-manager/default.nix

```diff
@@ -11,8 +11,8 @@ let
 
   pathStr = if path == null then "" else path;
 
-  nixos-option =
-    callPackage (pkgs.path + "/nixos/modules/installer/tools/nixos-option") { };
+  nixos-option = pkgs.nixos-option or (callPackage
```

Comment:
With the additional parentheses it also now evaluates correctly on all versions.